nomad/structs/host_volumes.go

```diff
@@ -221,6 +217,14 @@ func (hv *HostVolume) CanonicalizeForCreate(existing *HostVolume, now time.Time)
 		hv.CapacityBytes = 0 // returned by plugin
 		hv.HostPath = ""     // returned by plugin
 		hv.CreateTime = now.UnixNano()
+
+		if len(hv.RequestedCapabilities) == 0 {
+			hv.RequestedCapabilities = []*HostVolumeCapability{{
+				AttachmentMode: HostVolumeAttachmentModeFilesystem,
+				AccessMode:     HostVolumeAccessModeSingleNodeWriter,
+			}}
```

Comment:
I am curious about implications of this in the TF provider -- it could either be a computed property or explicit, and I don't recall how easy it is to account for an explicit option overwriting a computed one, if it gets added to the TF resource.

Reply:
If I understand what I've seen so far correctly (which isn't a given :grin:), you can set a schema field as `Optional: true` _and_ `Computed: true` and so long as the computed value comes back the same if unchanged, that works fine.